src/MutagenConfigManipulator.ts

```diff
@@ -78,7 +78,7 @@ export class MutagenConfigManipulator {
         if (fs.existsSync(file)) {
             fs.unlinkSync(file);
         } else {
-            throw new MutagenConfigNotFoundError(file + ' does not exist');
```

Comment:
Maybe better to leave control of how to handle this situation to the caller of the function?